lib/SIL/SILConstants.cpp

```diff
@@ -105,6 +114,8 @@ SymbolicValue::Kind SymbolicValue::getKind() const {
   case RK_Function:     return Function;
   case RK_Aggregate:    return Aggregate;
   case RK_Enum:         return Enum;
+  case RK_EnumWithPayload:
```

Comment:
My OCD asks for you to please make all the cases consistent  :-)  Please either shift all the returns over a bit and make them line up, or put the return on the next line for each one.

Reply:
Applied clang-format on this function, which had the latter effect.